app/forms/openid_connect_logout_form.rb

```diff
@@ -58,7 +82,18 @@ def identity_from_payload(payload)
     AgencyIdentityLinker.sp_identity_from_uuid_and_sp(uuid, sp)
   end
 
+  def id_token_hint_or_client_id_present
+    return if client_id.present? || id_token_hint.present?
+
+    errors.add( # rubocop:disable IdentityIdp/ErrorsAddLinter
+      :base,
+      t('openid_connect.logout.errors.no_client_id_or_id_token_hint'),
+    )
```

Comment:
```suggestion
    errors.add(
      :base,
      t('openid_connect.logout.errors.no_client_id_or_id_token_hint'),
      key: :no_client_id_or_id_token_hint,
    )
```
we want to keep this linter enabled and add a `key`

Reply:
It applies to two different attributes, which is why I ended up using `base` - would you recommend setting up a different key? I will add that adding a key makes the error message render in a really confusing way on the page.

Reply:
ohhhh I see, that's helpful thanks!

Reply:
fwiw the correct key is `type` I think - `key` still threw the linter error